src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -294,26 +304,32 @@ class DemuxFastqs
  @arg(doc="Platform model to insert into the group header (ex. miseq, hiseq2500, hiseqX)") val platformModel: Option[String] = None,
  @arg(doc="Comment(s) to include in the merged output file's header.", minElements = 0) val comments: List[String] = Nil,
  @arg(doc="Date the run was produced, to insert into the read group header") val runDate: Option[Iso8601Date] = None,
- @arg(doc="Output gzipped FASTQs (`.fastq.gz`) instead of BAM files") val outputFastqs: Boolean = false,
+ @arg(doc="The type of outputs to produce.") var outputType: OutputType = DemuxFastqs.DefaultOutputType,
```

Comment:
Is this a `var` for a reason?

Reply:
Also, is the default used in multiple places?  I think it's less clear than just having the value right here.